react/components/InputCurrency/index.js

```diff
@@ -45,20 +45,31 @@ class InputCurrency extends Component {
       currency: currencyCode,
     })
 
-    const formattedParts = formatter.formatToParts(baseNumber)
-    const prefix = !formattedParts.map(part => part.type).indexOf('currency')
+    const formattedParts =
+      formatter.formatToParts && formatter.formatToParts(baseNumber)
+    const prefix = formattedParts
+      ? !formattedParts.map(part => part.type).indexOf('currency')
+      : true
     const [currencySymbol] = formattedParts
-      .filter(part => part.type === 'currency')
-      .map(part => part.value)
+      ? formattedParts
+          .filter(part => part.type === 'currency')
+          .map(part => part.value)
+      : ['¤']
```

Comment:
🤨

Reply:
I don't think many people would understand this symbol. How about a `$`?

Reply:
I really don't know. I had never even seen it, but according to google it's commonly used for generic currency.